mobile-app/app/screens/AppNavigator/screens/Loans/components/LoanCardsV2.tsx

```diff
@@ -0,0 +1,202 @@
+import { useRef } from "react";
+import {
+  ThemedFlashList,
+  ThemedIcon,
+  ThemedText,
+  ThemedTextV2,
+  ThemedViewV2,
+} from "@components/themed";
+import { tailwind } from "@tailwind";
+import { translate } from "@translations";
+import NumberFormat from "react-number-format";
+import { View } from "react-native";
+import { getNativeIcon } from "@components/icons/assets";
+import {
+  LoanToken,
+  LoanVaultActive,
+  LoanVaultState,
+} from "@defichain/whale-api-client/dist/api/loan";
+import {
+  NavigationProp,
+  useNavigation,
+  useScrollToTop,
+} from "@react-navigation/native";
+import { ActivePrice } from "@defichain/whale-api-client/dist/api/prices";
+import { useSelector } from "react-redux";
+import { RootState } from "@store";
+import { vaultsSelector } from "@store/loans";
+import { getPrecisedTokenValue } from "@screens/AppNavigator/screens/Auctions/helpers/precision-token-value";
+import { getActivePrice } from "../../Auctions/helpers/ActivePrice";
+import { LoanParamList } from "../LoansNavigator";
+import { LoanActionButton } from "./LoanActionButton";
+
+interface LoanCardsProps {
+  loans: LoanToken[];
+  testID?: string;
+  vaultId?: string;
+  vaultExist: boolean;
+}
+
+export interface LoanCardOptions {
+  loanTokenId: string;
+  symbol: string;
+  displaySymbol: string;
+  price?: ActivePrice;
+  interestRate: string;
+  onPress: () => void;
+  testID: string;
+  isBorrowHidden: boolean;
+}
+
+export function LoanCardsV2(props: LoanCardsProps): JSX.Element {
+  const ref = useRef(null);
+  useScrollToTop(ref);
+  const navigation = useNavigation<NavigationProp<LoanParamList>>();
+  const vaults = useSelector((state: RootState) => vaultsSelector(state.loans));
+  const activeVault = vaults.find(
+    (v) =>
+      v.vaultId === props.vaultId && v.state !== LoanVaultState.IN_LIQUIDATION
+  ) as LoanVaultActive;
+  return (
+    <>
+      <ThemedFlashList
+        estimatedItemSize={2}
+        contentContainerStyle={tailwind("pt-4 pb-2")}
+        data={props.loans}
+        ref={ref}
+        numColumns={2}
+        renderItem={({
+          item,
+          index,
+        }: {
+          item: LoanToken;
+          index: number;
+        }): JSX.Element => {
+          return (
+            <View style={{ flexBasis: "100%" }}>
```

Comment:
Hmm this seems to be the source of this issue raised by @JoshuaC817 https://github.com/DeFiCh/wallet/pull/3513#issuecomment-1253422583
perhaps w-full? or reduce percentage of flexBasis for web?

Reply:
seem like this issue on the web itself and kinda known issue. Will recheck possible fix

Reply:
FlashList's rendered size is not usable warning. Something about this
https://shopify.github.io/flash-list/docs/known-issues/